src/wtc-gallery-component.js

```diff
@@ -52,10 +54,25 @@ class Gallery extends ElementController {
       this.element.appendChild(this.prevBtn);
     }
 
-    // Add pause-on-hover mouse events:
+    // Add pause-on-hover pointer events. Including a fallback to mouse events.
     if (this.options.pauseOnHover) {
-      element.addEventListener('mouseenter', this.pause.bind(this), false);
-      element.addEventListener('mouseleave', this.resume.bind(this), false);
+      if (window.PointerEvent) {
+        element.addEventListener('pointerover', this.pause.bind(this), false);
+        element.addEventListener('pointerout', this.resume.bind(this), false);
+      } else {
+        element.addEventListener('mouseenter', this.pause.bind(this), false);
+        element.addEventListener('mouseleave', this.resume.bind(this), false);
+      }
+    }
+
+    // Add "draggable" events
+    if (this.options.draggable) {
+      this.dragStartX = null;
+
+      element.addEventListener('mousedown', this.draggablePointerDown.bind(this), false);
```

Comment:
We should add a note to the documentation for this that this controller should never be stored in an immutable data structure as doing so will lead to mem leaks because of these bindings.

Reply:
Added a brief note under **Caveats** in the Readme